### DISCUSSION TITLE
有没有办法更换应用商店的软件源

### DISCUSSION THREAD

**Discussion by xiexiaosheng** [1Panel-dev/1Panel#7540] [General]
有没有办法把应用商店的软件源从apps.1panel.hk切换至apps-assets.fit2cloud.com
新装的机器默认软件源是apps.1panel.hk，应用太少了

**Comment by wanghe-fit2cloud** [comment]
卸载重装国内版本即可，文档：
https://1panel.cn/docs/installation/online_installation/
